scripts/release.sh

```diff
@@ -11,6 +11,6 @@ yarn install
 # ./scripts/generate_dev_docs.sh
 
 # Remove beta once we are out of it
-npx lerna publish --conventional-commits --yes --preid 'beta' --@zendesk:registry=https://npmjs.com
+npx lerna publish --conventional-commits --yes --preid 'beta' --@zendesk:registry=https://registry.npmjs.com
```

Comment:
Does this actually work? I thought it should be `--registry=https://registry.npmjs.org`

Reply:
According to https://stackoverflow.com/a/67691043 one should be able to do this.

Reply:
I've seen things that work for npm/yarn but not lerna. Please test it out. Thanks.

Reply:
Yes, you are correct. This doesn't work.